src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,86 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int WORK_DATE = 0;
+    private static final int EMPLOYEE_NAME = 1;
+    private static final int WORK_HOUR = 2;
+    private static final int RATE = 3;
+    private static final int SALARY_FOR_PERIOD = 1;
+    private static final String DATE_PATTERN = "d.MM.yyyy";
+    private int reportDataFreeIndex;
+    private final StringBuilder stringBuilder = new StringBuilder();
```

Comment:
if your class is utilized several times such field instantiation breaks its logic.

so, StringBuilder instantiation should be moved inside methods.